config/locales/en.yml

```diff
@@ -1,19 +1,13 @@
+---
 en:
   date:
     formats:
-      default:
-        "%m/%d/%Y"
```

Comment:
@teoljungberg I think it's silly that it puts the values on newlines when it's unnecessary. Also it's inconsistent with the style of all the other locale files and other yml files. Is that a fair reason?